### PR TITLE
Update `framework.exceptions` XML and PHP configuration examples

### PR DESCRIPTION
**Pull request by MatTheCat**
https://github.com/symfony/symfony/pull/48259 doc PR

Fix https://github.com/symfony/symfony-docs/issues/17609

### CODE REVIEW

File: reference/configuration/framework.rst [hunk 1]
```diff
@@ -3596,13 +3595,9 @@ exceptions that match the given exception class:
...
-                ->log_level('debug');
-
-            $framework
-                ->exceptions(BadRequestHttpException::class)
-                ->status_code(422);
+            $framework->exception(BadRequestHttpException::class)
+                ->logLevel('debug')
+                ->statusCode(422)
             ;
         };
```
Comment (mdoutreluingne) [inline reference/configuration/framework.rst:3603]: Deprecated directive for version 6.3 is missing. Can you add it?

Comment (MatTheCat) [reply reference/configuration/framework.rst:3603]: Is that really necessary since the deprecated format was never documented?

Comment (mdoutreluingne) [reply reference/configuration/framework.rst:3603]: I agree, it is not necessary

**Review by mdoutreluingne — APPROVED** [review]

**Comment by OskarStark** [comment]
Thanks Mathieu.

**Pull request merged**